pkg/handlers/v1/fetchbyip.go

```diff
@@ -0,0 +1,78 @@
+package v1
+
+import (
+	"context"
+	"net"
+
+	"github.com/asecurityteam/ipam-facade/pkg/domain"
+	"github.com/asecurityteam/ipam-facade/pkg/logs"
+)
+
+// IPAddressQuery contains an IP address on which to search for physical assets.
+type IPAddressQuery struct {
+	IPAddress string `json:"ipAddress"`
+}
+
+// PhysicalAssetDetails provides the response structure for PhysicalAsset records returned from storage.
+type PhysicalAssetDetails struct {
+	IP            string `json:"ip"`
+	ResourceOwner string `json:"resourceOwner"`
+	BusinessUnit  string `json:"businessUnit"`
+	Tags          tags   `json:"tags"`
```

Comment:
question: why make these nested as tags vs. top level attributes? is that an IPAM/D42 thing?

Reply:
The thinking here was that these fields are less important, but could be useful in a context beyond our immediate use case, so we demoted them to a nested structure, similar to what we are using for the asset inventory API.

I am completely open to alternative reasoning, however, so please let me know what you're thinking!